department-of-veterans-affairs/vets-website
# VAOS no facilities supported alert

## Description
This PR:
- Updates the NoValidFacilitiesV2 component to list out nearby unsupported facilities
- Updates some of our mock helpers to provide easier ways of setting common data
- Updates the Amputation care type of care to have no supported facilities
- Fixes a heading order issue for error message on facility page

How to test:
- Make sure you have feature toggle va_online_scheduling_flat_facility_page turned on
- Open new appointment flow: http://localhost:3001/health-care/schedule-view-va-appointments/appointments/new-appointment
- Choose Amputation care
- See alert message on VA facility page


## Testing done
Unit testing and local testing

## Screenshots

Limited to two facilities with address to sort by:
![Screen Shot 2021-01-22 at 3 48 54 PM](https://user-images.githubusercontent.com/634932/105756040-1a4d6400-5f1a-11eb-8c7d-cade5a95c759.png)

Alphabetical, limited to 5:
![Screen Shot 2021-01-22 at 3 48 34 PM](https://user-images.githubusercontent.com/634932/105756042-1ae5fa80-5f1a-11eb-8af9-562e16e64fa0.png)

## Acceptance criteria
- [ ] Designs match the specs
- [ ] If user has an address on file, show nearest possible facilities
- [ ] If facilities are shown, list 2 max
- [ ] If user does not have an address on file show facilities in alphabetical order, max 5

## Definition of done
